ygot/struct_validation_map_test.go

```diff
@@ -1680,9 +1675,44 @@ func TestMergeStructs(t *testing.T) {
 		inB: &validatedMergeTestWithSlice{
 			SliceField: []*validatedMergeTestSliceField{{String("chinook-single-hop")}},
 		},
-		wantErr: "error merging b to new struct: source and destination lists must be unique",
+		wantErr: "source and destination lists must be unique",
 	}}
 
+	for _, tt := range tests {
```

Comment:
I suggest that we make `tests` be defined outside of this function and then have a `TestMergeStructs` and `TestMergeStructsInto` -- just so we continue to have a 1:1 mapping between function-under-test and testcase.

Reply:
I agree. I think somehow the same set of tests should be run for both functions, so that any new function added automatically applies to both.

Reply:
Sounds good -- split the functions up and pulled out a shared test cases slice for each to use.